src/lib.rs

```diff
@@ -9,6 +9,7 @@ Repo: https://github.com/tox-rs/tox
 #![doc(html_logo_url = "https://raw.githubusercontent.com/tox-rs/logo/master/logo.png")]
 // Remove it when it will be fixed in nom parser
 #![allow(clippy::redundant_closure)]
+#![allow(clippy::drop_copy)]
```

Comment:
I'd rather see `let _ = copy` than `drop(copy)`.

Reply:
My preference is exactly opposite.

Reply:
Clippy's preferences are opposite to yours.

Reply:
Fortunately, clippy can be fixed with a single directive.

Reply:
This is not a place to fight clippy. If you want to change the code style, please do it in another PR.

Reply:
<3